scripts/convert_lit_checkpoint.py

```diff
@@ -196,6 +196,19 @@ def maybe_unwrap_state_dict(lit_weights: Dict[str, torch.Tensor]) -> Dict[str, t
     return lit_weights.get("model", lit_weights)
 
 
+def check_conversion_supported(lit_weights: Dict[str, torch.Tensor]) -> None:
+    weight_names = {wk.split(".")[-1] for wk in lit_weights}
+    # LoRA or QLoRA
+    if any("lora" in wn for wn in weight_names):
+        raise NotImplementedError(f"Model weights must be merged using `lora.merge_lora_weights()` before conversion.")
```

Comment:
@carmocca would you prefer for this to be an Exception instead of NotImplementedError?

Reply:
Maybe ValueError

Reply:
implemented in [9ab402f](https://github.com/Lightning-AI/lit-gpt/pull/397/commits/9ab402f44120917cd6f0f4b049fcad1be62be24d)